app/models/manageiq/providers/vmware/infra_manager/host_esx.rb

```diff
@@ -2,6 +2,30 @@ class ManageIQ::Providers::Vmware::InfraManager::HostEsx < ManageIQ::Providers::
   supports :refresh_advanced_settings
   supports :refresh_firewall_rules
   supports :refresh_logs
+  supports :reboot do
+    validate_active_with_power_state(:reboot, "on")
+  end
+  supports :shutdown do
+    validate_active_with_power_state(:shutdown, "on")
+  end
+  supports :standby do
+    validate_active_with_power_state(:standby, "on")
+  end
+  supports :enter_maint_mode do
```

Comment:
any reason to not spell it out?

```suggestion
  supports :enter_maintenance_mode do
```

Reply:
Wanted to keep these feature names the same https://github.com/ManageIQ/manageiq/pull/21847/files#diff-9daa2c365cb8a7cb177648fe53854dae6588080e089b01333768fcb5da8ff4aeL256